app/services/search_service.rb

```diff
@@ -196,12 +196,13 @@ def textSearch_questions(srchString)
     sresult
   end
 
-  # Search nearby nodes with respect to given latitude and longitude
-  def nearbyNodes(srchString)
+  # Search nearby nodes with respect to given latitude, longitute and tags
+  def tagNearbyNodes(srchString, tagName)
     sresult = DocList.new
-    coordinates = srchString.split(",")
-    lat = coordinates[0]
-    lon = coordinates[1]
+    lat, lon =  srchString.split(',')
+
+    tagList = textSearch_tags(tagName)
```

Comment:
Hi @stefannibrasil !
This looks great ! 
Just one thing --- should we not show nodes with tagName which our near the given Latitude and Longitude . 
This will add all nodes having tag =  tagName , right ?
What do you think ?